packages/xo-server/src/_handleBackupLog.mjs

```diff
@@ -1,3 +1,19 @@
+import moment from 'moment-timezone'
+import humanFormat from 'human-format'
+
+let vmBackupInfo = {}
+
+function sendToNagios(app, vmBackupInfo) {
+  app.sendPassiveCheck(
+    {
+      message: JSON.stringify(vmBackupInfo),
+      status: 0,
+    },
+    vmBackupInfo.vmName.toLowerCase(),
+    vmBackupInfo.jobName
+  )
```

Comment:
This is async function, it should always be `await`-ed or `.catch()` any errors.

```suggestion
  try {
    await app.sendPassiveCheck(
      {
        message: JSON.stringify(vmBackupInfo),
        status: 0,
      },
      vmBackupInfo.vmName.toLowerCase(),
      vmBackupInfo.jobName
    )
  } catch (error) {
    warn('sendToNagios', { error })
  }
```

Use `@xen-orchestra/log`.

Reply:
I did put the function in a try/catch statement. Is the ```await``` keyword still necessary? If so, I have to change the ```sendToNagios``` function to an async function, right?

Reply:
Thx!